spring-boot-project/spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -522,97 +520,98 @@ override those defined in lower locations).
 NOTE: You can also <<boot-features-external-config-yaml, use YAML ('.yml') files>> as
 an alternative to '.properties'.
 
-If you don't like `application.properties` as the configuration file name you can switch
-to another by specifying a `spring.config.name` environment property. You can also refer
-to an explicit location using the `spring.config.location` environment property
-(comma-separated list of directory locations, or file paths).
+If you do not like `application.properties` as the configuration file name, you can switch
+to another file name by specifying a `spring.config.name` environment property. You can also refer
+to an explicit location by using the `spring.config.location` environment property
+(a comma-separated list of directory locations or file paths). The following example
+shows how to specify a different file name:
 
 [indent=0]
 ----
 	$ java -jar myproject.jar --spring.config.name=myproject
 ----
 
-or
+The following example shows how to specify two locations:
 
 [indent=0]
 ----
 	$ java -jar myproject.jar --spring.config.location=classpath:/default.properties,classpath:/override.properties
 ----
 
 WARNING: `spring.config.name` and `spring.config.location` are used very early to
-determine which files have to be loaded so they have to be defined as an environment
-property (typically OS env, system property or command line argument).
+determine which files have to be loaded, so they must be defined as an environment
+property (typically an OS environment variable, a system property, or a command line argument).
 
-If `spring.config.location` contains directories (as opposed to files) they should end
-in `/` (and will be appended with the names generated from `spring.config.name` before
+If `spring.config.location` contains directories (as opposed to files), they should end
+in `/` (and be appended with the names generated from `spring.config.name` before
```

Comment:
I changed the line to "in `/` (and, at runtime, be appended with the names generated from `spring.config.name` before".

The idea behind removing "will" is to be consistent about tense and to avoid making promises. When we need to spell out what happens at compile-time/design-time versus what happens at runtime, we should do so explicitly and pitch "will" in the bin.